picoquic/packet.c

```diff
@@ -253,7 +254,9 @@ int picoquic_parse_packet_header(
                      ph->ptype = picoquic_packet_1rtt_protected_phi1;
                  }
 
+		 ph->hasspinbit = 1;
                  ph->spin = (bytes[0] >> 2) & 1;
+		 ph->VEC = bytes[0] & 0x03 ;
 
```

Comment:
Careful there. This branch of code supports both draft-11 and draft-12. You cannot use the VEC functionality in draft-11, the two bits are defined for another usage. So you need to test the version,  set VEC to zero if draft 11, and to the 2 bits if draft-12 and higher.

Reply:
The idea was that these internal state variables are harmless in draft-11, since the serialization code ignores them, and  that indeed support for earlier versions would soon disappear. That's why I felt it was OK to spare the extra version-switch complexity. OK to do it if you prefer.

Reply:
Please comment on these considerations ; I'll comply.